.pre-commit-config.yaml

```diff
@@ -1,8 +1,35 @@
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: no-commit-to-branch
+        args: ['--branch', 'main']
```

Comment:
`main` is already included in the default branch detection, so you can omit `args` if you want to simplify.

Reply:
I like the idea of having it explicit so that it's clear what it's doing to someone who doesn't know that main is already included.